resources/content/schema/user/user-auth.json

```diff
@@ -98,12 +98,13 @@
         "credential.publicValue" : "r",
         "credential.secretValue" : "r",
 
-        "host" : "r",
+        "host" : "ru",
         "host.agentId" : "r",
         "host.computeTotal" : "r",
         "host.info" : "r",
         "host.physicalHostId" : "r",
         "host.zoneId" : "ru",
```

Comment:
Did you verify whether we actually want zoneId to updatable? It was previously not updatable since host was updateable, but now that you've made host updateable, it is.